content/developers/apps/building-github-apps/authenticating-with-github-apps.md

```diff
@@ -101,33 +101,42 @@ puts jwt
 Here is a similar script for generating a JWT in Python. Note you will have to use `pip install jwt` in order to use this script. This script will prompt you for the location of your PEM file, or you can pass it as an inline argument when you execute the script. Replace `YOUR_APP_ID` with the ID of your app. Make sure to enclose the value in single quotes.
 
 ```python{:copy}
+#!/usr/bin/env python3
 import jwt
 import time 
 import sys
 
+
 # Get PEM file path
 if len(sys.argv) > 1:
     pem = sys.argv[1]
 else:
     pem = input("Enter path of private PEM file: ")    
 
+# Get the App ID from input / defaultß
+if len(sys.argv) > 2:
+    app_id = sys.argv[2]
+else:
+    app_id = input("Enter your APP ID: ") 
+
 # Open PEM
-with open(pem, 'r') as pem_file:
-    signing_key = pem_file.read()
+with open(pem, 'rb') as fh:
+    signing_key = jwt.jwk_from_pem(fh.read())
```

Comment:
nit: I'm going to keep this variable as `pem_file` instead of `fh` to be more descriptive
```suggestion
with open(pem, 'rb') as pem_file:
    signing_key = jwt.jwk_from_pem(pem_file.read())
```